src/Uno.UI/Microsoft/UI/Xaml/Controls/CommandBarFlyout/CommandBarFlyoutCommandBarTemplateSettings.cs

```diff
@@ -0,0 +1,331 @@
+﻿using Windows.Foundation;
+using Windows.UI.Xaml;
+
+namespace Microsoft.UI.Xaml.Controls.Primitives;
+
+/// <summary>
+/// Provides calculated values that can be referenced as TemplatedParent sources when defining
+/// templates for a CommandBarFlyout control. Not intended for general use.
+/// </summary>
+internal class CommandBarFlyoutCommandBarTemplateSettings : DependencyObject
```

Comment:
This looks public (and **sealed**) in WinUI
https://learn.microsoft.com/en-us/windows/winui/api/microsoft.ui.xaml.controls.primitives.commandbarflyoutcommandbartemplatesettings?view=winui-2.8